core/src/main/java/nl/nn/adapterframework/jdbc/dbms/GenericDbmsSupport.java

```diff
@@ -413,10 +417,49 @@ public String getBooleanValue(boolean value) {
 		return (""+value).toUpperCase();
 	}
 
+	protected ISqlTranslator createTranslator(String source, String target) throws JdbcException {
+		return new SqlTranslator(source, target);
+	}
+
 	@Override
 	public void convertQuery(QueryExecutionContext queryExecutionContext, String sqlDialectFrom) throws SQLException, JdbcException {
```

Comment:
Als ik echt mag zeuren vind ik dat we ergens onze dialecten hard moeten defineren. Nu gebeurd dat half in een spring bestand en kan iedereen elk dialect opgeven dat hij of zij wilt.